src/main/java/cinema/model/MovieSession.java

```diff
@@ -0,0 +1,65 @@
+package cinema.model;
+
+import java.time.LocalDateTime;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "movie_session")
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne
+    private Movie movie;
+    @ManyToOne
+    @JoinColumn(name = "cinema_hall")
```

Comment:
Why do you need this annotation? OR Why do you use this annotation only to `cinemaHall`?

Reply:
Fixed